### DISCUSSION TITLE
How to stop the server?

### DISCUSSION THREAD

**Discussion by shaggyrec** [uNetworking/uWebSockets.js#389] [Q&A] [answered]
Does TemplatedApp have the `stop` method? 
How to stop server from code?

**Comment by hst-m** [comment, marked as answer]
[us_listen_socket_close](https://unetworking.github.io/uWebSockets.js/generated/index.html#us_listen_socket_close) stops the server from listening: [example](https://github.com/uNetworking/uWebSockets.js/blob/master/examples/GracefulShutdown.js#L17-L33) shutdown

**Comment by shaggyrec** [comment]
OMG. Sorry
